internal/services/appservice/linux_web_app_resource_test.go

```diff
@@ -1130,7 +1130,7 @@ func TestAccLinuxWebApp_vNetIntegration(t *testing.T) {
 				),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("virtual_network_subnet_id"),
```

Comment:
is the API not returning this value for some reason? if so we can pull it across from state in the read?

this is usually an indicator of a bug

Reply:
The API will return the vNET ID. The issue is this property can be set via another resource `azurerm_app_service_virtual_network_swift_connection`, TF would write it back to state, causing diff since user doesn't specifies it in web_app resource block.

Reply:
the we should make the property computed instead of just ignoring it? 

either way this is not the correct way to fix it

Reply:
Thanks @katbyte for the comments. We are not able to make this property computed as the change won't be detected if the property is set in the resource. We need to educate the user to use ignore_change for this property. We've made a clear guidance in the docs, I'll close this pr. Thanks again for the review!